benchmarks/maxtext_xpk_runner.py

```diff
@@ -336,6 +400,41 @@ def build_user_command(
   return command
 
 
+def _get_pathways_proxy_flags(wl_config: WorkloadConfig):
+  """Get the pathways proxy flags for the workload and removes any extras."""
+  # Add in the xla flags alongside the proxy flags from the pathways config.
+  pw_config = wl_config.pathways_config
+
+  # Get proxy and xla flag string from model config
+  proxy_flags_string = pw_config.proxy_flags
+  xla_flags_string = wl_config.model.xla_flags
+
+  # Split both proxy_flags_string and xla_flags_string into lists of flags
+  proxy_flags_list = proxy_flags_string.strip().split()
+  xla_flags_list = xla_flags_string.strip().split()
+
+  # Combine the two lists of flags into a single list
+  proxy_flags = proxy_flags_list + xla_flags_list
+
+  # Remove the flags that are specified to be removed.
+  if (
+      wl_config.model.pathways_xla_flag_options
+      and model_configs.REMOVE in wl_config.model.pathways_xla_flag_options
```

Comment:
Trying to understand what `model_configs.REMOVE` would look like. Assuming that it is a list of flags to be removed, why would it be present in `wl_config.model.pathways_xla_flag_options` ?

Reply:
Great question! There may be some XLA flags that are present in mcjax that we don't want in the Pathways run, so we remove them.